custom/icds_reports/models.py

```diff
@@ -1,10 +1,15 @@
 from __future__ import absolute_import
 
 from __future__ import unicode_literals
+
+import architect
+from datetime import datetime
 from dateutil.relativedelta import relativedelta
 from django.db import connections, models
+from jsonfield import JSONField
```

Comment:
Can you use the official `JSONField` instead of the unofficial library? https://docs.djangoproject.com/en/2.0/ref/contrib/postgres/fields/#jsonfield